lib/coarsecoarse_op.cu

```diff
@@ -3,19 +3,39 @@
 #include <gauge_field.h>
 
 #define COARSECOARSE
-#ifdef GPU_MULTIGRID
 #include <coarse_op.cuh>
-#endif
+
 namespace quda {
 
-#ifdef GPU_MULTIGRID
+  static CloverField* dummyClover()
```

Comment:
I assume this dummy field is just to enable using pre-compiled routines? If so (or if not), can you please add a comment noting the reason?

Reply:
Exactly that is the reason.

Reply:
Can add comment.